src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileStatusService.scala

```diff
@@ -16,10 +16,13 @@ class FileStatusService(fileStatusRepository: FileStatusRepository)(implicit val
   }
 
   def allChecksSucceeded(consignmentId: UUID): Future[Boolean] = {
+    val getChecksumMatchStatus = fileStatusRepository.getFileStatus(consignmentId, ChecksumMatch)
+    val getAvStatus = fileStatusRepository.getFileStatus(consignmentId, Antivirus)
+    val getFfidStatus = fileStatusRepository.getFileStatus(consignmentId, FFID)
```

Comment:
Can these be put inside the for loop similarly to how you've done it for the others, for consistency?